airflow-core/src/airflow/ui/src/router.tsx

```diff
@@ -54,7 +54,7 @@ import { XCom } from "src/pages/XCom";
 
 import { Configs } from "./pages/Configs";
 import { Security } from "./pages/Security";
-import { queryClient } from "./queryClient";
+import { client } from "./queryClient";
```

Comment:
useQueryClient ?

Reply:
This one we want to use the queryClient we create but I wanted the name to be different so we don't accidentally import our local `queryClient` instead of using `useQueryClient()`